src/documentation/ntfs2gtfs.md

```diff
@@ -90,7 +90,7 @@ stop_desc | no | comments.txt, comment_links.txt | comment_name | The value of `
 
 GTFS field | Required | NTFS file | NTFS field | Note
 --- | --- | --- | --- | ---
-route_id | yes | trips.txt | route_id |
+route_id | yes | trips.txt and routes.txt | line_id corresponding to the route_id |
```

Comment:
The specs have been updated : https://github.com/CanalTP/navitia_model/blob/master/src/documentation/ntfs2gtfs.md#routestxt
I think this modification is not necessary